packages/block-editor/src/components/block-list/use-block-props/index.js

```diff
@@ -106,6 +110,8 @@ export function useBlockProps(
 				enableAnimation:
 					! isTyping() &&
 					getGlobalBlockCount() <= BLOCK_ANIMATION_THRESHOLD,
+				isExplodedMode: __unstableGetEditorMode() === 'exploded',
```

Comment:
I think we should think through this name a bit more. Maybe "zoom-out" or "zoomed-out"?